package-lock.json

```diff
@@ -2549,4 +2549,4 @@
       "integrity": "sha512-r3vXyErRCYJ7wg28yvBY5VSoAF8ZvlcW9/BwUzEtUsjvX/DKs24dIkuwjtuprwJJHsbyUbLApepYTR1BN4uHrg=="
     }
   }
-}
+}
```

Comment:
This file should be removed :)

Reply:
Okay @gs0510 i hope i have done that now

Reply:
Hello @gs0510 i've tried to revert the file to the original one but there is no change, I have tried many times to update the package-lock.json file to the one on the upstream main branch but it's still showing that nothing has been changed.
Please look at the line 2552 and you'll see that it's the same with the one of my pull request.